Tools/Scripts/run-jsc-stress-tests

```diff
@@ -2567,7 +2582,8 @@ def handleCollectionDirectory(collection)
         $benchmarkDirectory = $collection
         allJSFiles($collection).each {
             | path |
-            
+
+            next if not $isWasmPlatformx and isWasmTest(path)
```

Comment:
typo: `$isWasmPlatformx` should be `$isWasmPlatform`

Reply:
Thanks.  Will fix.